docs/080-breaking-changes.rst

```diff
@@ -155,3 +156,18 @@ Interface Changes
   Use the "compact AST" (``--ast-compact--json`` resp. ``AST``) as replacement.
 
 * The old error reporter (``--old-reporter``) has been removed.
+
+
+How to update your code
+=======================
+
+- If you rely on wrapping arithmetic, surround each operation with ``unchecked { ... }``.
+- Optional: If you use SafeMath or a similar library, change ``x.add(y)`` to ``x + y``, ``x.mul(y)`` to ``x * y`` etc.
+- Add ``pragma abicoder v1;`` if you want to stay with the old ABI coder.
+- Optionally remove ``pragma experimental ABIEncoderV2`` or ``pragma abicoder v2`` since it is redundant.
```

Comment:
Should mention that actually `pragma experimental ABIEncoderV2` is invalid.

Reply:
Do we have a conversion step in solidity-upgrade for this?

Reply:
There are no conversion steps in `solidity-upgrade` for 0.8.0.

Reply:
Didn't we somewhere say that "solidity-upgrade does a lot of the manual work"?

Reply:
Well nobody implemented anything and this was never prioritized.

Reply:
I mean we have such a line in the docs or the release notes :)

Reply:
We removed that line!